<?xml version="1.0"?>
<launch>
    <arg name="use_jax" default="true" />
    <rosparam file="$(find angle_aware_control)/config/central.yaml" command="load" ns="" />

    <group if="$(arg use_jax)">
        <node pkg="angle_aware_control" type="psi_generator.py" name="psi_generator" output="screen" required="true" />
    </group>
    <group unless="$(arg use_jax)">
        <node pkg="angle_aware_control" type="psi_generator_no_jax.py" name="psi_generator" output="screen" required="true" />
    </group>

</launch>